src/package.xml

```diff
@@ -113,6 +113,8 @@
         <members>UTIL_Describe_TEST</members>
         <members>UTIL_Namespace</members>
         <members>UTIL_Namespace_TEST</members>
+        <members>UTIL_Profile</members>
+        <members>UTIL_Profile_TEST</members>
```

Comment:
Why would these two be included now?

Reply:
Since these classes were added to the repo, the deployment fails without them in the package.

Reply:
Right, that makes sense. I'm just a little surprised the PR where they got included (https://github.com/SalesforceFoundation/HEDAP/pull/623/files) passed build with the classes being there but not listed in package.xml...